apps/labs/posts/intel-oneapi-msvc-spin-scipy.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Enhancing Developer Experience at SciPy - Intel oneAPI/MSVC Support and Migrating to spin'
+published: December 19, 2024
+authors: [gagandeep-singh]
+description: 'Highlights the work done to improve developer experience at SciPy, specifically on supporting Intel oneAPI/MSVC and spin'
+category: [Developer workflows]
+featuredImage:
+  src: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_featured.png
+  alt: 'SciPy logo'
+hero:
+  imageSrc: /posts/intel-oneapi-msvc-spin-scipy/scipy_logo_img_hero.png
+  imageAlt: 'SciPy logo'
+---
+
+In this blog post, we will be sharing the work we did to enhance developer experience at SciPy. We worked on two major things,
+
+1. Intel oneAPI and MSVC Support in SciPy
+2. Moving away from `dev.py` and migrating to `spin`.
+
+First, let me share the motivation behind these.
+
+Intel offers a complete toolkit for high performance computing (HPC), numerical computing use cases. It goes by the name of [oneAPI](https://www.intel.com/content/www/us/en/developer/tools/oneapi/overview.html). It provides, ICX (for C), ICPX (for C++), IFX (for Fortran) and Math Kernal Library (MKL for optimized math routines like BLAS, LAPACK, fast FFT, etc.) [[1]](https://en.wikipedia.org/wiki/Math_Kernel_Library). Since, SciPy caters to HPC and numerical computing use cases, so supporting building SciPy with Intel oneAPI is a natural requirement. In fact, users reported segmentation fault in `arpack` with `ifx` [[gh-20728]](https://github.com/scipy/scipy/issues/20728) and `scipy.linalg.solve` crash when used with Intel oneAPI on Windows 11 [[gh-21997]](https://github.com/scipy/scipy/issues/21997) are two such examples. Therefore we decided to work on this to enhance the experience of SciPy users who use Intel oneAPI as their toolkit. Regarding MSVC, since it is the de-facto compiler for C/C++ on Windows. So adding support for it is again a natural requirement and also a must.
```

Comment:
- I'd use "MKL (Math Kernel Library, ...)" rather than the other way around, since you're also using the abbreviations for ICX & co
- MSVC: we didn't actually add support for it though, so the rationale here is a little off. Try to rephrase to what we actually aimed to do.

Reply:
I think MSVC wasn't tested on CI before our work, right? On Windows we used clang for CI testing. So may be a better line can be, "Since it is the de-facto compiler for C/C++ on Windows, we added CI jobs for testing SciPy compilation with MSVC"